src/buttonGroup/buttonGroup.story.mdx

```diff
@@ -0,0 +1,73 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+import { Button, ButtonStatus } from '../button'
+import { ButtonGroup } from './index'
+
+<Meta title="Buttons|ButtonGroup" />
+
+# ButtonGroup
+
+### Stack
+
+<Preview>
+  <Story name="Stack">
+    <ButtonGroup>
+      <Button>Hello</Button>
+      <Button status={ButtonStatus.TERTIARY}>there</Button>
+    </ButtonGroup>
+  </Story>
+</Preview>
+
+### InLine
+
+<Preview>
+  <Story name="InLine">
+    <ButtonGroup isInline>
+      <Button>Hello</Button>
+      <Button status={ButtonStatus.TERTIARY}>there</Button>
+    </ButtonGroup>
+  </Story>
+</Preview>
+
+### Reverse
+
+<Preview>
+  <Story name="Reverse">
+    <ButtonGroup isReverse={true}>
+      <Button>Hello</Button>
+      <Button status={ButtonStatus.TERTIARY}>there</Button>
+    </ButtonGroup>
+  </Story>
+</Preview>
+
+## Specifications
+
+This component has two layout configurations based on small vs large screen size.
+
+On small devices button group become vertical and consists of Primary and Tertiary buttons. Buttons width are 100%.
+
+On large devices it's horizontal and consists of Primary and Secondary buttons placed inline with reduced spacing between them: 16pt.
+Each button occupy 50% of the available space.
+
+## Usage
+
+```jsx
+import { ButtonGroup } from '@blablacar/ui-library'
+<ButtonGroup>
+  <Button>Hello</Button>
+  <Button status={ButtonStatus.TERTIARY}>there</Button>
+</ButtonGroup>
+```
+
+<Props of={ButtonGroup} />
+
+## When should I use it?
+
+Use it at the end of a flow. Allows the member to make a final decision.
+
+- ✅ Always at the bottom of the screen
+- ✅ Can be sticky
+- ⛔️ Labels can't go on 2 lines. If too long we use "…"
+
+## TODO
+
+- [ ] Force button types
```

Comment:
Is it a Pixar todo or is it for us?

Reply:
It was there, I think we should remove the "TODOs"